assembly/frame/executive.ts

```diff
@@ -141,13 +141,13 @@ export namespace Executive {
             }
             
             const blockNumber = SystemStorageEntries.Number().get();
-            const blockHash = SystemStorageEntries.BlockHash().get(instantiate<BlockNumber>(blockNumber.unwrap() - 1));
+            const blockHash = SystemStorageEntries.BlockHash().get(instantiate<BlockNumber>(blockNumber.unwrap() - 2));
```

Comment:
seems strange

Reply:
Yeah. It was the fix for the signature verification. I suspect there's sth in the block initialisation. I think we can leave it for the next milestone